src/collective/cover/content.py

```diff
@@ -128,6 +135,33 @@ def set_tile_data(self, id, **data):
         data_mgr = ITileDataManager(tile)
         data_mgr.set(data)
 
+    def get_referenced_objects(self):
+        """Get referenced objects from cover object.
+
+        :returns: a set of objects referenced
+        :rtype: set of objects
+        """
+        refs = set()
+        for tile_uuid in self.list_tiles():
+            tile = self.get_tile(tile_uuid)
+            uuid = tile.data.get('uuid', None)
+            if uuid is not None:
+                refs |= set([uuidToObject(uuid)])
+            if IListTile.providedBy(tile):
+                uuids = tile.data.get('uuids', [])
+                if uuids is None:
+                    continue
+                for uuid in uuids:
+                    refs |= set([uuidToObject(uuid)])
+            elif IRichTextTile.providedBy(tile):
+                value = tile.data.get('text')
+                if value is None:
+                    continue
+                value = value.raw
+                links = extractLinks(value)
+                refs |= getObjectsFromLinks(self, links)
+        return refs
+
```

Comment:
yes, and integration test could be interesting here just to see if the references are correctly returned